paf-mvp-benchmark/K6/data.js

```diff
@@ -0,0 +1,81 @@
+export const unsignedPrefsMock = {
```

Comment:
It would be great to use typescript here and import the generated models.
This way, this `unsignedPrefsMock` could be typed, easier for maintenance.